internal/pkg/utils/slice.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+func Contains(haystack []string, needle string) bool {
+	for _, x := range haystack {
+		if x == needle {
+			return true
+		}
+	}
+	return false
+}
+
+func Remove(haystack []string, needle string) []string {
+	for i, x := range haystack {
+		if x == needle {
+			return append(haystack[:i], haystack[i+1:]...)
+		}
+	}
+	return haystack
+}
+
+func RemoveDuplicates(slice []string) []string {
+	m := make(map[string]bool)
```

Comment:
Unfortunate that we can't use set here because of circular imports

Reply:
Yeah, I was thinking about this too... I'll make a generic `types` package!

Reply:
(To avoid the circular import error, which I think we would get)